### PR TITLE
Move text-layer from experimental-layers to core-layers

### PR DESCRIPTION
**Pull request by gnavvy**
For #976

#### Change List
- Moved text-layer code and doc from experimental-layers to core-layers
- Moved text-layer related examples from experimental folder to core-layer folder
- Updated paths accordingly
(rendering test failed locally, tested on @jianhuang01's machine)

### CODE REVIEW

**Comment by gnavvy** [comment]
> And of course a render test would be awesome, maybe @jianhuang01 can assist here?

Rendering test does not work (large ∆ due to color difference) on my laptop for several test cases.
We have tested the code on Jian's machine and it worked well.

**Comment by ibgreen** [comment]
@jianhuang01 Maybe we can add a mode where we do only grayscale comparison to see if this makes the tests work on a wider range of machines?

**Comment by Pessimistress** [comment]
You may want to wait for https://github.com/uber/deck.gl/pull/1595 to land

**Comment by jianhuang01** [comment]
We already have a test case for text layer, and it runs well with this PR. @ibgreen Tested on @gnavvy 's machine, not only color is different, some lines are also broken into line segments.

File: website/contents/pages.js [hunk 1]
```diff
@@ -89,6 +89,13 @@ export const examplePages = generatePath([
           demo: 'ScatterplotDemo',
           code: getCodeUrl('examples/scatterplot')
         }
+      },
+      {
+        name: 'TextLayer',
```
Comment (jianhuang01) [inline website/contents/pages.js]: The example is still primitive, maybe hold it until it is better?

**Comment by ibgreen** [comment]
>  some lines are also broken into line segments.

That does not sound good. Maybe worth debugging to understand why this happens on some machines? 

Maybe a screenshot to help understand the severity of the issue?

**Comment by jianhuang01** [comment]
@ibgreen Will take a screenshot when getting a chance, the rendering result is flaky on @gnavvy 's machine, not 100% reproducible.

**Pull request merged**